dnf/cli/commands/module.py

```diff
@@ -342,10 +352,14 @@ def set_argparser(self, parser):
         narrows.add_argument('--all', dest='all',
                              action='store_true',
                              help=_("remove all modular packages"))
-
-        subcommand_help = [subcmd.aliases[0] for subcmd in self.SUBCMDS]
-        parser.add_argument('subcmd', nargs=1, choices=subcommand_help,
-                            help=_("Modular command"))
+        subcommand_choices = []
+        subcommand_help = [_('Following modular commands are available:')]
```

Comment:
I think this should read:
> **The** following modular commands are available:

But I find it a bit long, I'd consider either:
> A modular command:

Or even leave it out altogether, I think it's still clear in the help output. But either is fine, really.